felix/bpf/attach.go

```diff
@@ -82,13 +82,14 @@ func ListCalicoAttached() (map[string]EPAttachInfo, error) {
 	for _, p := range aTC {
 		if strings.HasPrefix(p.Name, "cali") {
 			info := ai[p.DevName]
-			if p.Kind == "tcx/egress" {
+			switch p.Kind {
```

Comment:
I like this kind of change. Could you perhaps filer those out and commit separalely?